content/_help/trouble-signing-in/face-or-touch-unlock._en.md

```diff
@@ -58,6 +67,20 @@ Go to [secure.login.gov](https://secure.login.gov/) and click on "Create an acco
 2. You will receive an email. Click on the link in the email to confirm your email address.
 3. Create a password. It must be at least 12 characters long.
 4. On the “Secure your account” page, select face or touch unlock.
-5. Enter a device nickname for face or touch unlock.
-6. Upon clicking the “continue” button, you will then be prompted to authenticate using face or touch unlock. 
-7. Follow the steps to set up another authentication method. 
+5. On the face or touch unlock set up page, click on the “continue” button. You will then be prompted to authenticate using face or touch unlock.
+    **Important**: Depending on your device, you may be prompted for additional information such as a PIN, pattern, or QR code. Follow the directions on your screen to finish setting up face or touch unlock. The following are screenshots of possible prompts you may encounter:
+
+    <div class="grid-row grid-gap">
+        <div class="grid-row">
+            <img alt="Example of a prompt that may be encountered upon setting up or using face or touch unlock: Screenshot of a screen lock PIN prompt" src="{{ site.baseurl }}/assets/img/help/face-touch-unlock/android-screen-lock.png" />
+        </div>
+        <div class="grid-row margin-y-5">
+            <img alt="Example of a prompt that may be encountered upon setting up or using face or touch unlock: Screenshot with a passkey message that allows users to choose between using iCloud keychain or or a phone or tablet" src="{{ site.baseurl }}/assets/img/help/face-touch-unlock/iphone-screen-lock.png" />
+        </div>
+        <div class="grid-row">
+            <img alt="Example of a prompt that may be encountered upon setting up or using face or touch unlock: Screenshot of a QR code" src="{{ site.baseurl }}/assets/img/help/face-touch-unlock/passkey-screen-shot.png" />
+        </div>
+    </div>
```

Comment:
If we're not using the grid anymore, can we simplify to use Markdown syntax instead of HTML? We could apply classes to the images to make sure they're stacked with a gap.

Untested, but I'm assuming something like this could work:

```suggestion
    ![Example of a prompt that may be encountered upon setting up or using face or touch unlock: Screenshot of a screen lock PIN prompt]({{ site.baseurl }}/assets/img/help/face-touch-unlock/android-screen-lock.png){: .display-block}
    ![Example of a prompt that may be encountered upon setting up or using face or touch unlock: Screenshot with a passkey message that allows users to choose between using iCloud keychain or or a phone or tablet]({{ site.baseurl }}/assets/img/help/face-touch-unlock/iphone-screen-lock.png){: .display-block margin-top-5}
    ![Example of a prompt that may be encountered upon setting up or using face or touch unlock: Screenshot of a QR code]({{ site.baseurl }}/assets/img/help/face-touch-unlock/passkey-screen-shot.png){: .display-block margin-top-5}
```

Reply:
yeah this is a good way to go about this

Reply:
Looks like the images are appearing broken in the Cloud.gov Pages preview, not really sure why.

Could we also apply this to the other languages as well?

Reply:
I messed around in [eab6ca8](https://github.com/18F/identity-site/pull/1196/commits/eab6ca85a0e5c2487489e4802e161f0beb7ed8ac) and it looks like cloud.gov did not like the `site.baseurl` part of the image link. Not sure why I'm writing this comment...

Reply:
Now that I think about this some more, I think it is because everything in markdown is contained within the file structure itself?